textproc/mandoc/Portfile

```diff
@@ -20,8 +19,8 @@ long_description    mandoc is a suite of tools compiling mdoc, \
 
 master_sites        https://mandoc.bsd.lv/snapshots/
 
-checksums           rmd160 0155d0670421c37aa79c1887ecab3904236907cd \
-                    sha256 0b0c8f67958c1569ead4b690680c337984b879dfd2ad4648d96924332fd99528
+checksums           rmd160 5e3702e49a70a270184132bf08f891eb12100ea2 \
+                    sha256 24eb72103768987dcc63b53d27fdc085796330782f44b3b40c4660b1e1ee9b9c
```

Comment:
We now usually add a "sizes" field here as well.

Reply:
@janstary It's a shame holding up a simple and useful update for five days over something as simple as a size field. Can you push a fix for that and we'll merge?